### DISCUSSION TITLE
Path aliases in *.data.ts. Error "Cannot find module"

### DISCUSSION THREAD

**Discussion by said-m** [vuejs/vitepress#2279] [Q&A] [answered]
Hi, i need your help to solve the problem… getting an error when ts path alias resolves from `*.data.ts` files.
Here an example: https://stackblitz.com/edit/vite-j8rvru?file=docs/blog/posts.data.ts

* `constants.ts`
* `**/helper.ts` - it has import with path alias: `~/constants`
* `**/some.md` - imports `./helper` and everything is ok
* `**/some.data.ts` - imports `./helper` and it fails

is it an issue or am i doing something wrong in the configuration?

**Comment by said-m** [comment]
@brc-dd hi! sorry for interrupting. if you have time, please check if it is a vitepress issue. 
vitepress@1.0.0-alpha.73 installed.
if necessary, I will provide more detailed information on reproducing the problem.

**Reply by brc-dd** [reply, marked as answer]
Created an issue from this: #2289. Track it instead. I'll get back to you there once I get some time.

**Reply by brc-dd** [reply]
The current behavior is similar to how vite loads it config file. I don't think it respects alias in that as well.
